packages/frontend/src/features/sqlRunner/components/ContentPanel.tsx

```diff
@@ -81,120 +81,18 @@ export const ContentPanel: FC<Props> = ({
-                                onClick={() => {
-                                    if (!sql) return;
-                                    runSqlQuery({
-                                        sql,
-                                    });
-                                }}
-                            >
-                                Run query
-                            </Button>
-                        </Tooltip>
-                    </Group>
-                </Group>
-            </Paper>
-            <Paper
-                ref={inputSectionRef}
-                shadow="none"
-                radius={0}
-                p="none"
-                withBorder
-                style={{ flex: 1 }}
-            >
-                <Box
-                    sx={{
-                        position: 'absolute',
-                        height: inputSectionHeight,
-                        width: inputSectionWidth,
-                    }}
-                >
-                    <SqlEditor sql={sql} onSqlChange={setSql} />
-                </Box>
-            </Paper>
-            <ResizableBox
-                height={resultsHeight}
-                minConstraints={[50, 50]}
-                maxConstraints={[Infinity, maxResultsHeight]}
-                resizeHandles={['n']}
-                axis="y"
-                handle={
-                    <Divider
-                        h={3}
-                        bg="gray.3"
-                        pos="absolute"
-                        top={-2}
-                        left={0}
-                        right={0}
-                        sx={{
-                            cursor: 'ns-resize',
-                        }}
-                    />
-                }
-                style={{
-                    position: 'relative',
-                    display: 'flex',
-                    flexDirection: 'column',
-                }}
-                onResizeStop={(e, data) => setResultsHeight(data.size.height)}
-            >
+            <Tooltip.Group>
                 <Paper shadow="none" radius={0} px="md" py="sm" withBorder>
                     <Group position="apart">
-                        <Group spacing="xs">
-                            <Title order={5} c="gray.6">
-                                Results/Chart panel
-                            </Title>
-                            {isLoading && <Loader size="xs" />}
-                        </Group>
-
+                        <Title order={5} c="gray.6">
+                            SQL panel
+                        </Title>
                         <Group spacing="md">
                             <Tooltip
+                                key={String(isResultsHeightMoreThanHalf)}
                                 variant="xs"
                                 label={
-                                    isResultsHeightMoreThanHalf
+                                    !isResultsHeightMoreThanHalf
```

Comment:
Just checking if this should be inverted?

If the results are move than half, then I want to be able to collapse... or not?

Reply:
Oops. Didn't mean to change that 🤨